libraries/AP_Baro/AP_Baro_MS5611.h

```diff
@@ -88,19 +88,19 @@ class AP_Baro_MS56XX : public AP_Baro_Backend
     AP_SerialBus *_serial;
 
     /* Asynchronous state: */
-    volatile bool            _updated;
+    volatile bool            _updated = false;
```

Comment:
adding zero initialisers like this increases the size of the PX4 firmware without actually achieving anything. I started to do that myself a while ago then noticed the sharp increase in PX4 firmware size. Without our new/malloc wrapper guaranteeing zero init I think we should leave these out